debugging/kernel-logger/kp_kernel_logger.cpp

```diff
@@ -137,18 +137,27 @@ extern "C" void kokkosp_end_parallel_reduce(const uint64_t kID) {
 }
 
 extern "C" void kokkosp_begin_fence(const char* name, const uint32_t devID, uint64_t* kID) {
-	*kID = uniqID++;
+		
+	 if (std::strstr(name, "Kokkos Profile Tool Fence"))
+           {
+              *kID = std::numeric_limits<uint64_t>::max();
+           }
+        else
+          {
+          *kID = uniqID++;
 
-	printf("KokkosP: Executing fence on device %d with unique execution identifier %llu\n",
-		devID, *kID);
+        printf("KokkosP: Executing fence on device %d with unique execution identifier %llu\n",
+                devID, *kID);
 
-	int level = kokkosp_print_region_stack();
-	kokkosp_print_region_stack_indent(level);
-	
-	printf("%s\n", name);
+        int level = kokkosp_print_region_stack();
+        kokkosp_print_region_stack_indent(level);
+
+        printf("%s\n", name);
+		
```

Comment:
Is it just me or is a closing brace missing here, so that this wouldn't even compile??

Reply:
Yes, looks like older commit from this past week  - fixing